lib/core/versions/0.5.0/BatchWriter.ts

```diff
@@ -63,7 +63,13 @@ export default class BatchWriter implements IBatchWriter {
     console.info(`Wrote anchor file ${anchorFileAddress} to content addressable store.`);
 
     // Anchor the 'anchor file hash' on blockchain.
-    await this.blockchain.write(anchorFileAddress);
+    //
+    // NOTE: We are passing 0 for the fee parameter in this version. The fee parameter was
+    // introduced after this version had shipped. This 'write' function is only ever called
+    // for the latest blockchain time which means that this previous version will never be
+    // called and passing a fake value like 0 is ok.
```

Comment:
Nit: just want to call out that technically, the last/previous version of the `BatchWriter` can be active for a period of time before the blockchain time crosses over to the next version.